openff/toolkit/utils/ambertools_wrapper.py

```diff
@@ -69,10 +70,7 @@ def __init__(self):
                 f"available. {self._toolkit_installation_instructions}"
             )
 
-        # TODO: More reliable way to extract AmberTools version
-        out = subprocess.check_output(["antechamber", "-L"])
-        ambertools_version = out.decode("utf-8").split("\n")[1].split()[3].strip(":")
-        self._toolkit_version = ambertools_version
+        self._toolkit_version = get_ambertools_version()
```

Comment:
For reference, source code is here: https://github.com/openforcefield/openff-utilities/blob/ee9f969c252d02548c08537fb34cb713923eef70/openff/utilities/provenance.py#L39-L42